spring-integration-zeromq/src/main/java/org/springframework/integration/zeromq/ZeroMqUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.zeromq;
+
+import org.zeromq.ZMQ;
+
+/**
+ * Module that wraps common methods of ZeroMq integration classes
+ *
+ * @author Alessio Matricardi
+ *
```

Comment:
Sorry, I might was not right: the `@since` is required for new classes. And after that for new `public` methods added to existing classes.
So, bring back `@since 6.4` over here, please